plugins/octant/cmd/antrea-octant-plugin/main.go

```diff
@@ -42,28 +39,36 @@ const (
 	overviewTitle   = "Overview"
 	controllerTitle = "Antrea Controller Info"
 	agentTitle      = "Antrea Agent Info"
-	versionCol      = "Version"
-	podCol          = "Pod"
-	nodeCol         = "Node"
-	serviceCol      = "Service"
-	crdCol          = "Monitoring CRD"
-	subnetCol       = "NodeSubnet"
-	bridgeCol       = "OVS Bridge"
-	podNumCol       = "Local Pod Num"
-	heartbeatCol    = "Last Heartbeat Time"
+	traceflowTitle  = "Trace Flow"
```

Comment:
Maybe rename traceflowTitle to "Traceflow Info" so it will be consistent with controllerTitle and agentTitle

Reply:
Title changed.